hail/src/main/scala/is/hail/expr/ir/EmitCodeBuilder.scala

```diff
@@ -149,4 +149,7 @@ class EmitCodeBuilder(val emb: EmitMethodBuilder[_], var code: Code[Unit]) exten
   def strValue(r: Value[Region], t: PType, code: Code[_]): Code[String] = {
     StringFunctions.boxArg(EmitRegion(emb, r), t)(code).invoke[String]("toString")
   }
+
+  // for debugging
+  def println(cString: Code[String]) = this += Code._println(cString)
```

Comment:
maybe have this take varargs of Code[String] and do `Code._printlns(strings:_*)`, I use that much more than the single arg (you get concat for free)

Reply:
Ah, didn't even know about that one, good idea.